lib/lib.d.ts

```diff
@@ -1005,7 +1005,7 @@ interface ReadonlyArray<T> {
       */
     toString(): string;
     /**
-      * Returns a string representation of an array. The elements are converted to string using thier toLocalString methods.
```

Comment:
files under lib/* are not meant to be manually changed. they are generated by `jake LKG`. so please revert these changes

Reply:
@mhegazy thanks for the review. src/lib/* is the correct place right? And do I need to run `jake LKG`  and commit the changes? it generates quite a big changeset.